src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Proxy/SessionHandlerProxyTest.php

```diff
@@ -179,8 +179,25 @@ public static function provideNativeSessionStorageHandler()
             [new SessionHandlerProxy(new StrictSessionHandler(new \SessionHandler()))],
         ];
     }
+
+    public function testCreateSid()
+    {
+        $mock = $this->createMock(SessionIdSessionHandler::class);
+        $mock->expects($this->once())
+            ->method('create_sid');
+
+        $proxy = new SessionHandlerProxy($mock);
+        $proxy->create_sid();
+
+        $this->proxy->create_sid();
+        $this->addToAssertionCount(1);
```

Comment:
let's assert that both return the same value?

Reply:
`$proxy->create_sid()` would return the mock’s `create_sid` and `$this->proxy->create_sid()` would return the result of `session_create_id()` so they’re not supposed to return the same value?

Reply:
as the mock implements the interface, the proxy is expected to return the sid created by the mock. Your test does not enforce that.

Reply:
Oh okay I added an assertion :+1: